rhtconsulting/tssc-python-package
# Feature/napsspo 996 : use a settings.xml for all maven steps to allow for disconnected.

Goal:  To create a settings.xml file for each of the maven step_implementers:
(1)  push_artifacts
(2)  package
(3)  uat
(4)  unit_test
Changes:
- use the 'new' sops for passwords...
- create a utils/maven to create settings.xml
- update uat to use -s settings.xml
- update package to use -s settings.xml
- REWORK push artifacts to use -s settings.xml 
  remove user/password
  add repo
Note that test/step_implementer/unit_test/test_maven.py was renamed to test_maven_unit_test.py because of the newly created util/tests/test_maven.py (unique file names for the tests.)
